easybuild/easyconfigs/f/fabio/fabio-0.10.2-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'fabio'
```

Comment:
@jhein32 We try to stick to "official" software names, this should be `FabIO`.

Likewise for `PyOpenCL`.

Fixed in https://github.com/jhein32/easybuild-easyconfigs/pull/7

Reply:
I was actually wondering about the rules around casing and figured, if I were to use your casing, it would not find the correct tarball.   I didn't get through to things like `SOURCELOWER_TAR_GZ` which fix that very issue.